test/e2e/data/infrastructure-outscale/cluster-template-with-clusterclass.yaml

```diff
@@ -128,7 +128,7 @@ spec:
           name: "{{ ds.meta_data.local_hostname }}"
           kubeletExtraArgs:
             cloud-provider: external
-            provider-id: osc://'{{ ds.meta_data.instance_id }}'
+            provider-id: aws:///'{{ ds.meta_data.instance_id }}'
```

Comment:
this is not for the PR, watchout during the commit !

Reply:
It is remove